as_export.py

```diff
@@ -59,14 +60,14 @@ def _run(self):
         if self.update_time:
             self.store_last_export_time()
         elif self.digital_only or self.digital_resource_id:
-            self.export_digital_objects(resource=self.digital_resource_id)
+            self.export_digital_objects(self.client, resource=self.digital_resource_id)
```

Comment:
You don't need to pass the client as an argument here since it's already defined on `self` (which is passed into all the methods of the class)